CHANGES.md

```diff
@@ -6,6 +6,43 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
+## Release 0.322.0
+* Provide row count on the dbapi cursor for DML queries.
+  ([#325](https://github.com/trinodb/trino-python-client/issues/325))
+
+## Release 0.321.0
```

Comment:
I suppose this 0.321.0 entry should be removed after rebasing on master.

Reply:
yeah I saw that after but didn't have time to fix. Will fix shortly.